runtime/src/main/scala/tailcall/runtime/transcoder/Config2Blueprint.scala

```diff
@@ -302,13 +304,22 @@ object Config2Blueprint {
     private def updateModifyField(
       field: Field,
       bField: Blueprint.FieldDefinition,
+      typeInfo: Type,
     ): TValid[String, Option[Blueprint.FieldDefinition]] = {
       field.modify match {
         case Some(ModifyField(None, Some(true)))    => TValid.none
         case Some(ModifyField(Some(newName), None)) =>
-          val resolverPath = if (bField.resolver.isEmpty) List("value", bField.name) else List()
-          val resolver     = Lambda.identity[DynamicValue].path(resolverPath: _*).toDynamic
-          TValid.succeed(appendResolver(bField, resolver).copy(name = newName)).some
+          val interfaceWithSameField = config.graphQL.types.toList.filter(_._2.interface.isDefined)
+            .filter(interface => typeInfo.implements.toList.flatten.contains(interface._1))
+            .find(interface => interface._2.fields.find { case (iFieldName, _) => iFieldName == bField.name }.isDefined)
+          interfaceWithSameField match {
+            case Some(value) => TValid
+                .fail(s"Cannot use modify on field ${bField.name} because it is implemented from interface ${value._1}")
```

Comment:
```suggestion
                .fail(s"Implemented fields are unmodifiable")
```
More concise error message IMO. The field name and type name is already available in the trace.